packages/config/src/handlers/updateQuestion/handler.ts

```diff
@@ -53,17 +53,23 @@ export default async function updateQuestion(
   return h.response(existingQuestion).code(201)
 }
 
-const messageDescriptorSchema = Joi.object({
-  id: Joi.string().required(),
-  defaultMessage: Joi.string(),
-  description: Joi.string()
+const messageSchema = Joi.array().items({
+  lang: Joi.string(),
+  descriptor: Joi.object({
+    id: Joi.string().required(),
+    defaultMessage: Joi.string(),
+    description: Joi.string()
+  })
```

Comment:
This is being used in separate places. Maybe we can use a common one?

Reply:
Yeah. I have to import it from createQuestions. Thanks.